templates/server.conf.erb

```diff
@@ -55,14 +65,20 @@ ssl_certfile: <%= scope['pulp::db_ssl_certfile'] %>
 verify_ssl: <%= scope['pulp::db_verify_ssl'] %>
 ca_path: <%= scope['pulp::db_ca_path'] %>
 <% end -%>
-
+<% if scope['pulp::db_unsafe_autoretry'] -%>
+unsafe_autoretry: <%= scope['pulp::db_unsafe_autoretry'] %>
+<% end -%>
+<% if scope['pulp::db_write_concern'] != :undef-%>
```

Comment:
@ehelms Cant I argue that in our params.pp  we are exclusively marking it "undef" , why do i need to check for each
```
 unless [nil, :undefined, :undef, ''].
```